### ISSUE TITLE
VB line commit should add missing argument parentheses

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn#36545]
**Version Used**: 16.2 Preview 2

:link: https://developercommunity.visualstudio.com/content/problem/567910/vs2019-intellisense-worst-than-vs2010.html

**Steps to Reproduce**:

1. Type the following VB code:

    ```vb
    Console.WriteLine "Text"
    ```

2. Press <kbd>Enter</kbd>

**Expected Behavior**:

Parentheses are automatically added:

```vb
Console.WriteLine("Text")
```

**Actual Behavior**:

The text is not changed, and error BC30800 is reported saying:

> Method arguments must be enclosed in parentheses.

**Comment by dddns** [comment]
Thank you for this and the previous bug.
Will monitor to track changes.

**Comment by dddns** [comment]
@sharwell for this fix, the expected behavior like vs2010, is to add the parentesis like vs2010 when enter is pressed or CTRL+s (save)

**Comment by dddns** [comment]
@sharwell I just updated to 16.2.0 and this still happens/not fixed.

**Comment by dddns** [comment]
Hi,
Nock Nock, no fix for this yet?

**Comment by CyrusNajmabadi** [comment]
@dddns There is no one assigned to fix this. It's in the "help wanted" category.  Meaning: if you'd like to see this fixed, your contributions to help this project would always be appreciated.  Thanks! :)

**Comment by dddns** [comment]
@CyrusNajmabadi , more? It was me who reported this, they have a video with that bug explanation.

**Comment by CyrusNajmabadi** [comment]
> more?

Sure :)   As i mentioned, there is no one assigned to it and there are no votes for it.  So it's unlikely to be fixed unless someone particularly feels like going and fixing it themselves.  The best course of action if you'd like to see this change in a future version is just to submit a PR for it yourself :)

**Comment by dddns** [comment]
@sharwell already detected the bug/regression, i wont do nothing else.

**Comment by CyrusNajmabadi** [comment]
Closing out due to no feedback and no movement from community on contributing here.
